text/0000-add-scratch-stack.md

```diff
@@ -0,0 +1,159 @@
+# Create static stack
+
+## Summary
+
+The Paketo stacks maintainers should create and maintain a stack (called
+"static") that can run statically-linked applications (e.g. golang, Rust).
+The Paketo builder maintainers will make a buildpackless builder for this
```

Comment:
@paketo-buildpacks/builders-maintainers any objections to creating and maintaining a new buildpackless builder?

Reply:
I think that if we are going to build out a static builder at all, we would also want a buildpackless builder for consistency sake. It sounds like to support Go we need a builder therefore we should probably have a buildpackless builder.

Reply:
@ForestEckhardt just to be clear, your comment above sounds like a tacit approval of supporting a jammy-static-buildpackless-builder, yes?

Reply:
Yes, I approve of the buildpackless builder